api/src/resources/comment.rs

```diff
@@ -114,12 +114,12 @@ pub struct SyncCommentsResponse {
     pub unchanged: usize,
 }
 
-#[derive(Debug, Clone, Deserialize, Serialize, PartialEq, Eq)]
```

Comment:
Removing `Eq` because floats in Rust

Reply:
Nevermind this, I noticed we already use `ordered_float` and `NotNan` for this reason.